src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -332,7 +332,7 @@ gl_sarray activity_classifier::predict(gl_sframe data,
   }
 
   // Bind the data to a data iterator.
-  std::unique_ptr<data_iterator> data_it = create_iterator(data, false);
+  std::unique_ptr<data_iterator> data_it = create_iterator(data, false, false);
```

Comment:
Nit: for boolean args, I find it really helps readability to add an in-line comment indicating what the argument name is. For example:
```
std::unique_ptr<data_iterator> data_it = create_iterator(data, /* requires_labels */ false, /* is_train */ false);
```